libs/application/api-template-utils/src/lib/types.ts

```diff
@@ -19,3 +19,7 @@ export type SendEmail = { type: 'sendEmail'; template: SendMailOptions }
 export type ApplicationAPITemplateAction =
   | SendEmail
   | AssignApplicationThroughEmail
+
+export enum PDF_TYPES {
```

Comment:
A bit weird to have this type located here, while the api-template-utils doesn't do anything regarding PDF. I would define it locally in your app instead and if it's need to be shared between libraries and app then better to define inside shared/tyoes

Reply:
Totally agree, will move 👍